app/helpers/application_helper.rb

```diff
@@ -14,9 +14,4 @@ def round_meters_up_to_5_mi(meters)
     (meters / five_miles_in_meters).ceil * 5
   end
 
-  def you_or_spouse(intake)
-    return "you or your spouse" if intake.filing_joint_yes?
-
-    "you"
-  end
```

Comment:
this is also replaced with i18n pluralization rules in views. embedding string fragments like this inside of longer phrases tends to be a no-no for i18n.